base/assets/gm4/lang/es_es.json

```diff
@@ -64,15 +73,50 @@
     "block.gm4.ender_hopper": "Tolva del End",
     "block.gm4.master_crafter": "Mesa de Trabajo de maestro artesano",
     "block.gm4.treasure_chest": "Cofre del Tesoro",
+    "block.gm4.liquid_tank.empty": "Tanque Vacío",
+    "block.gm4.liquid_tank.water": "Tanque de Agua",
+    "block.gm4.liquid_tank.lava": "Tanque de Lava",
+    "block.gm4.liquid_tank.milk": "Tanque de Leche",
+    "block.gm4.liquid_tank.mushroom_stew": "Tanque de Sopa de Champiñones",
+    "block.gm4.liquid_tank.rabbit_stew": "Tanque de Estofado de Conejo",
+    "block.gm4.liquid_tank.beetroot_soup": "Tanque de Sopa de Remolacha",
+    "block.gm4.liquid_tank.experience": "Tanque de Experiencia",
+    "block.gm4.liquid_tank.fire_resistance": "Tanque de Poción de Resistencia al Fuego",
+    "block.gm4.liquid_tank.floating": "Tanque de Poción de Levitación",
+    "block.gm4.liquid_tank.harming": "Tanque de Poción de Daño",
+    "block.gm4.liquid_tank.healing": "Tanque de Poción de Curación",
+    "block.gm4.liquid_tank.invisibility": "Tanque de Poción de Invisibilidad",
+    "block.gm4.liquid_tank.leaping": "Tanque de Poción de Salto",
+    "block.gm4.liquid_tank.luck": "Tanque de Poción de Suerte",
+    "block.gm4.liquid_tank.night_vision": "Tanque de Poción de Visión Nocturna",
+    "block.gm4.liquid_tank.poison": "Tanque de Poción de Veneno",
+    "block.gm4.liquid_tank.regeneration": "Tanque de Poción de Regeneración",
+    "block.gm4.liquid_tank.slow_falling": "Tanque de Poción de Caída lenta",
+    "block.gm4.liquid_tank.slowness": "Tanque de Poción de Lentitud,
+    "block.gm4.liquid_tank.strength": "Tanque de Poción de Fuerza",
+    "block.gm4.liquid_tank.strong_harming": "Tanque de Poción de Daño (Fuerte)",
+    "block.gm4.liquid_tank.strong_healing": "Tanque de Poción de Curación (Fuerte)",
+    "block.gm4.liquid_tank.strong_leaping": "Tanque de Poción de Salto (Fuerte)",
+    "block.gm4.liquid_tank.strong_poison": "Tanque de Poción de Veneno (Fuerte)",
+    "block.gm4.liquid_tank.strong_regeneration": "Tanque de Poción de Regeneración (Fuerte)",
+    "block.gm4.liquid_tank.strong_slowness": "Tanque de Poción de Lentitud (Fuerte)",
+    "block.gm4.liquid_tank.strong_strength": "Tanque de Poción de Fuerza (Fuerte)",
+    "block.gm4.liquid_tank.strong_swiftness": "Tanque de Poción de Rapidez (Fuerte)",
+    "block.gm4.liquid_tank.strong_turtle_master": "Tanque de Poción del maestro Tortuga (Fuerte)",
+    "block.gm4.liquid_tank.swiftness": "Tanque de Poción de Rapidez",
+    "block.gm4.liquid_tank.turtle_master": "Tanque de Poción del maestro Tortuga,
```

Comment:
Missing quote
```suggestion
    "block.gm4.liquid_tank.turtle_master": "Tanque de Poción del maestro Tortuga",
```

Reply:
Fixed w/ the ''Mini-update 2 (Spanish Translations)'' commit.